.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Bug report
+about: Create a report to help us fix or improve an issue
+---
+
+**This is a bug report for the `@clr` Angular or UI versions of the design system.
+For the web-component implementation of Clarity (`@cds`), visit [vmware-clarity/core](https://github.com/vmware-clarity/core).**
+
+## Describe the bug
+
+_A clear and concise description of what the bug is, and the conditions when it occurs._
+
+## How to reproduce
+
+_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._
+
+Steps to reproduce the behavior:
+
+1.  Go to '...'
+2.  Click on '....'
+3.  Scroll down to '....'
+4.  See error
+
+## Expected behavior
+
+_A clear and concise description of what you expected to happen._
+
+## Versions
+
+**Clarity version:**
+
+- [ ] v12.x
+- [ ] v13.x
+- [ ] Other:
+
+**Framework:**
+
+- [ ] Angular
+- [ ] React
```

Comment:
do we need to have framework here?

Reply:
If it's ng-clarity, the only framework should be Angular. So I don't think this section is necessary.

Reply:
You're right, there's no reason to even bring these files over because they won't be used. I guess I just got carried away syncing everything up. I've removed these templates from this PR.